src/component/page/gjennomforing-page/GjennomforingListePage.module.scss

```diff
@@ -6,7 +6,14 @@
 }
 
 .seksjonGjennomforinger {
-  min-width: 600px;
+  @media (min-width: 600px) {
+    min-width: 600px;
+  }
+
+  @media(max-width: 600px) {
+    flex-grow: 1;
+  }
+
```

Comment:
Tror jeg ville skrevet dette om til å kun bruke mobile-first istedenfor å bruke teknikker fra både desktop-first og mobile-first. Selv om vi ikke skal designe for mobil, så er det greit å designe for små skjermer først. En annen liten endring er å bytte ut flex-grow med width: 100%. Begge fungerer, men flex-grow er avhengig av at komponenten ovenfor bruker flex, som er lett å glemme når man refaktorerer.
```
width: 100%;

  @media(min-width: 600px) {
    max-width: 600px;
  }
```

Reply:
Enig i design for små skjermer først, men er ikke enig at ting skal kunne vokse ut av parenten sin, så jeg ville mest sannsynlig brukt flex-grow. Men kan sikkert endre til width: 100% om det er det som er mest brukt i appen

Reply:
Men denne fiksen vil jo ikke fikse problemet, siden width vil være satt uansett. så med den css'en vil den ta hele skjermen. Jeg beholder det som det er